rust/server/src/bin/protoapp-server.rs

```diff
@@ -35,7 +36,7 @@ async fn run() -> anyhow::Result<()> {
         .map_err(|e| anyhow::anyhow!("unable to parse config file: {}", e))?;
 
     inject_secrets(&mut config)?;
-    server::run(config).await;
+    server::run(config).await?;
```

Comment:
I'm surprised that this snuck in.  I though the compiler warned when results were not consumed. Perhaps not with async...